content/docs/contributing/code-of-conduct.md

```diff
@@ -5,25 +5,10 @@ description: Before you join the O3DE project, make sure you understand the cond
 weight: 600
 ---
 
-## CNCF Community Code of Conduct v1.0
+## Community Code of Conduct v1.0
 
 Other languages available:
```

Comment:
Since you removed the list of languages, I assume you want to review this line, too?

Reply:
Thank you for flagging Willow. Yes, that should be removed as well.